packages/main/src/Table.js

```diff
@@ -468,6 +489,8 @@ class Table extends UI5Element {
 		this.visibleColumnsCount = this.visibleColumns.length;
 
 		this._allRowsSelected = selectedRows.length === this.rows.length;
+
+		this._previousFocusedRow = this._previousFocusedRow || this.rows[0] || null;
```

Comment:
the `|| null` part seems redundant. If `this.rows[0]` is falsy, it will be `undefined` and this is what will be returned.. Or do you specifically need `null` here?

Reply:
I see `null` as being the intentional absence of a value, for example when there are no rows to begin with. This seemed somewhat strange to me, but I found it in one of the testing samples:

Check the second table in https://sap.github.io/ui5-webcomponents/master/playground/main/pages/TableGrowingWithButton/

However, for the purpose of checking whether `_previousFocusedRow` is truthy, `undefined` will do just fine.